vitest-web.config.ts

```diff
@@ -16,6 +16,9 @@ export default defineConfig({
     exclude: [
       'src/lib/poll/poll.test.ts',
       'src/lib/cryptography/login.test.ts',
+      // Temporarily skipped until https://github.com/shapeshift/hdwallet/pull/666 goes in
+      'src/components/Modals/Send/hooks/useSendDetails/useSendDetails.test.tsx',
```

Comment:
unfortunate we have to do this, but I guess we aren't in these hooks much these days. I tried to leverage the alias in vitest config, but didn't seem to work... feels like their _should_ be a way